src/chrome/content/rules/GLS.de.xml

```diff
@@ -1,5 +1,9 @@
 <!--
-
+    Nonfunctional subdomains:
+        - laden            -> wrong domain
+        - sharedichdrum    -> too long
```

Comment:
Ah, that does the "SSL_ERROR_RX_RECORD_TOO_LONG" actually mean? A cipher/tls version mismatch?